content/08-cr-features.md

```diff
@@ -602,14 +602,36 @@ There are three levels of null safety strictness:
 
 * Off Turn off null safety checks. Useful to selectively disable null safety for particular fields or expression.
 * Loose (Default.) Within an `if (<expr> != null)` condition, `<expr>` is considered safe even if it could be modified after the check.
-* Strict Full-scale null safety checking.
+* Strict Full-scale null safety checking for a single-threaded environment.
+* StrictThreaded Full-scale null safety checking for a multi-threaded environment.
```

Comment:
The actual cases should be visually separated from the text. I would say backticks and a colon.

 - `Off`: Turn off blah blah
 - `Strict`: Blah blah